.gitignore

```diff
@@ -6,3 +6,8 @@ _static
 _templates
 
 TODO
+*.pyc
```

Comment:
IMHO these all belong in your global gitignore, they're not specific to this project (which is why they weren't here before).

In case you don't already have one here's a sample of mine: https://github.com/Julian/dotfiles/blob/master/.config/git/ignore